types/redux-form/lib/actions.d.ts

```diff
@@ -18,8 +18,8 @@ export declare function arraySplice(form: string, field: string, index: number,
 export declare function arraySwap(form: string, field: string, indexA: number, indexB: number): FormAction;
 export declare function arrayUnshift(form: string, field: string, value: any): FormAction;
 export declare function autofill(form: string, field: string, value: any): FormAction;
-export declare function blur(form: string, field: string, value: any): FormAction;
-export declare function change(form: string, field: string, value: any): FormAction;
+export declare function blur(form: string, field: string, value: any, touch: boolean): FormAction;
+export declare function change(form: string, field: string, value: any, touch?: boolean, persistentSubmitErrors?: boolean): FormAction;
```

Comment:
I think these two parameters should be required too.

Reply:
Based on `redux-form` types, they are optional, that's why I kept them optional, to not confuse people

Reply:
Well that's an implementation detail.
I think we can take the best of both worlds if we changed `blur` action to make `touch` optional. This way there will be no breaking changes.

WDYT?

Reply:
> ... but I think we can take the best of both worlds if we changed `blur` function to make `touch` optional. This way there will be no breaking changes for both functions

Agreed

Reply:
Done